mitosheet/src/mito/components/taskpanes/Graph/graphUtils.tsx

```diff
@@ -487,4 +495,82 @@ export const openGraphSidebar = async (
             }
         })
     }
+}
+
+export const registerClickEventsForGraphElements = (graphOutput: GraphOutput, setSelectedGraphElement: React.Dispatch<React.SetStateAction<GraphElementType | null>>) => {
+    console.log('here')
+    if (graphOutput === undefined) {
+        return;
+    }
+    const div: any = document.getElementById(graphOutput.graphHTML.split('id="')[1].split('"')[0])
+    if (div === null) {
+        return;
+    }
+    div.on('plotly_click', () => {
+        setSelectedGraphElement(null)
+    });
+    
+    // Main Title
+    const gtitle = div.getElementsByClassName('g-gtitle')[0]
```

Comment:
Ooo since we are already getting this object, can we use the default title in the x axis, y axis, and chart titles when you open the editor for the first time. 

Currently, if you have not yet manually set the titles, then the input just opens to a blank input.

Reply:
Ah yeah definitely